### PR TITLE
Add datadog_screenboard resource

### PR DESCRIPTION
**Pull request by oxlay**
This PR adds full support for Datadog screenboards.
All 19 widgets are supported.

Solves #52.

@masci I suppose you should be the one reviewing this?

### CODE REVIEW

File: vendor/vendor.json [hunk 1]
```diff
@@ -1,660 +1,660 @@
 {
-	"comment": "",
```
Comment (masci) [inline vendor/vendor.json]: can you keep the original indent so the diff is more visible?

File: website/docs/r/screenboard.html.markdown [hunk 2]
```diff
@@ -0,0 +1,635 @@
...
+	title = "Test Screenboard"
+	read_only = true
+
+	template_variable {
+		name    = "varname 1"
+		prefix  = "pod_name"
+		default = "*"
+	  }
+
+	  template_variable {
```
[the remark this answers is not in the document]
Comment (oxlay) [reply website/docs/r/screenboard.html.markdown]: You're right, should be better now 👍

File: datadog/resource_datadog_screenboard.go [hunk 3]
```diff
@@ -0,0 +1,1407 @@
...
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/kr/pretty"
+	datadog "github.com/zorkian/go-datadog-api"
+)
+
+// TODO:
```
[the remark this answers is not in the document]
Comment (oxlay) [reply datadog/resource_datadog_screenboard.go]: Yup, it was actually done anyway! Nice catch

File: vendor/vendor.json (hunk 1 above)
Comment (oxlay) [reply vendor/vendor.json]: 👍

**Comment by masci** [comment]
Thanks @oxlay this was a long awaited feature from a lot of users! Merging now and releasing on Monday.

**Pull request merged**